content/metadata.yaml

```diff
@@ -363,7 +363,7 @@ authors:
       - Writing - Original Draft
       - Writing - Review & Editing
     code of conduct:
-      confirmed: No
+      confirmed: Yes
     affiliations:
     - Biomedical Science, Midwestern University, Glendale, AZ, United States of America
     - Department of Neuroscience, Carleton University, Ottawa, Ontario, Canada
```

Comment:
@nafisajadavji would you be able to change the date below to 2020-11-11? Or else if you confirm your conflict of interest is still correct, I can make the change separately!

Reply:
@rando2 Sorry for the confusion, I am still figuring out Github. I think I did what you asked me to.

Reply:
You did!! Thank you so much! I know it's super tricky, especially with the .yaml file which isn't a totally common format off of GitHub!